court_scraper/platforms/wiscourts/pages/config.py

```diff
@@ -0,0 +1,17 @@
+from selenium.webdriver.common.by import By
+
+class WisconsinURLs:
+    
+    CASE_DETAILS = 'https://wcca.wicourts.gov/caseDetail.html'
+    SEARCH_PAGE = 'https://wcca.wicourts.gov/advanced.html'
+    
+class WisconsinVariables:
+    
+    SITEKEY = '6LeTUtAUAAAAALyw0QQWmOALx9f2eDS2Y23m6eVp'
```

Comment:
Is `SITEKEY` something we've requested as a particular user, i.e. something that will not be consistent/valid if another user runs the code?  Is it sensitive info that belongs in a config outside of version control?